salt/modules/netbsd_sysctl.py

```diff
@@ -149,7 +153,9 @@ def persist(name, value, config='/etc/sysctl.conf'):
         nlines.append("{0}\n".format(newline))
 
     with salt.utils.files.fopen(config, 'w+') as ofile:
-        ofile.writelines(nlines)
+        ofile.writelines(
+            salt.utils.data.decode(nlines)
```

Comment:
This needs to be an `encode`, and we should be opening the file using a mode of `'wb'`.